druid/src/widget/flex.rs

```diff
@@ -661,7 +655,12 @@ impl<T: Data> Widget<T> for Flex<T> {
                     *calculated_siz = kv.resolve(env);
                     major_non_flex += *calculated_siz;
                 }
-                Child::Flex(_, _, flex) | Child::FlexedSpacer(flex, _) => flex_sum += *flex,
+                Child::Flex {
```

Comment:
you can write this as, `Child::Flex { flex, .. } | Child::FlexedSpacer(flex, _) =>`.